cpp/client/ssl_client.cc

```diff
@@ -154,7 +147,7 @@ LogVerifier::LogVerifyResult SSLClient::VerifySCT(const string& token,
 int SSLClient::VerifyCallback(X509_STORE_CTX* ctx, void* arg) {
   VerifyCallbackArgs* args = reinterpret_cast<VerifyCallbackArgs*>(arg);
   CHECK_NOTNULL(args);
-  LogVerifier* verifier = args->verifier;
+  LogVerifier* verifier(args->verifier.get());
```

Comment:
That variable is a bit of a waste of time, no? :smiley: 

(don't have to fix it, your call)